### ISSUE TITLE
Plural glued to inline literal not detected

### ISSUE THREAD

**Issue by JulienPalard** [sphinx-contrib/sphinx-lint#18]
See: https://github.com/python/cpython/pull/32331

**Comment by ezio-melotti** [comment]
This would be useful, especially because in markdown both `` `a`s `` and ``` ``a``s ``` are correctly rendered as `a`s.

**Comment by ezio-melotti** [comment]
Isn't this already handled by this checker?
https://github.com/sphinx-contrib/sphinx-lint/blob/cecc01d0d6be9c6398558ffbed2de4b028590707/sphinxlint.py#L217-L222
